src/main/controllers/ApplicationDetailsController.ts

```diff
@@ -94,7 +95,7 @@ export default class ApplicationDetailsController {
       ...content,
       header,
       selectedApplication,
-      appContent: getTseApplicationDetails(selectedApplication, translations, downloadLink),
+      appContent: getTseApplicationDetails(selectedApplication, translations, downloadLink, req.url),
```

Comment:
Add welshEnabled feature flag

Reply:
Not sure if the welshEnabled flag should be added for this controller.  I made a change to the shared getTseApplicationDetails helper function, which required the url to be passed but the applicationDetailsController / respondToTribunalResponse weren't pages being that were translated in this pull request.